content/communities/maintaining-your-safety-on-github/unblocking-a-user-from-your-organization.md

```diff
@@ -33,6 +33,7 @@ If you selected a specific amount of time to block the user, they will be automa
 
 {% data reusables.profile.access_org %}
 {% data reusables.profile.org_settings %}
+{% data reusables.organizations.moderation-settings %}
 {% data reusables.organizations.block_users %}
```

Comment:
```suggestion
```

👋🏼  Hi @taku333, since we're using a new reusable (`moderation-settings.md`), we'll need to remove this outdated reusable here. Right now, this would keep the incorrect step in the instructions if we have both reusables.

@lecoursen's suggestions above apply to the new reusable. ✨

Reply:
I'm sorry to reply late. Thank you for confirmation.